### PR TITLE
junitrunner: Report suppressed and ignored tests as skipped.

### PR DESCRIPTION
**Pull request by benjaminp**
Fixes https://github.com/bazelbuild/bazel/issues/6688
(after java tools release)

### CODE REVIEW

File: src/java_tools/junitrunner/javatests/com/google/testing/junit/runner/model/AntXmlResultWriterTest.java [hunk 1]
```diff
@@ -115,6 +116,26 @@ public void testWallTimeAndMonotonicTimestamp() throws Exception {
...
+    Document document = parseXml(stringWriter.toString());
+    NodeList caseElems = document.getElementsByTagName("testcase");
+    assertThat(caseElems.getLength()).isEqualTo(2);
+    for (int i = 0; i < 2; i++) {
+      Element caseElem = (Element) caseElems.item(i);
+      NodeList skippedElems = caseElem.getElementsByTagName("skipped");
+      System.out.println(stringWriter.toString());
+      assertThat(skippedElems.getLength()).isEqualTo(1);
+    }
+  }
```
Comment (comius) [inline src/java_tools/junitrunner/javatests/com/google/testing/junit/runner/model/AntXmlResultWriterTest.java:137]: Please break the test into three blocks: arrange (setting things up), act (running), assert (verifying results). Separate each block with an empty line.
Remove System.out.println.

**Review by comius — APPROVED** [review]

**Comment by or-shachar** [comment]
YEY! :) ❤️

**Comment by Yannic** [comment]
@bazel-io flag

**Comment by Yannic** [comment]
Eh, wait. This should already be in Bazel 5.

**Pull request closed without merging**